Tests/conf.json

```diff
@@ -3835,6 +3835,7 @@
         {
             "integrations": "Azure Storage",
             "playbookID": "Azure Storage - Test",
+            "instance_names": "azure_storage_client_cred",
```

Comment:
please add the first instance_name  (before, it takes it as default because the secret has the integration name, now it considers only the new)